README.md

```diff
@@ -303,6 +303,25 @@ $ spago build --watch
 $ spago build --watch --clear-screen
 ```
 
+To run a command before a build you can use the `--before` flag, eg to clear the screen before a build:
+
+```bash
+$ spago build --watch --before clear
```

Comment:
To clear the screen we already have the `--clear-screen` option, could we pick an alternative example here? Or alternatively show them side-by-side

Reply:
Good point. I'll try to think of an alternative example.